### PR TITLE
review test

### PR DESCRIPTION
**Pull request by sstaar91**
review test

### CODE REVIEW

**Review by hsuj86 — COMMENTED** [review]
LGTM 👍🏻

**Review by hsuj86 — APPROVED** [review]
approve

**Review by hsuj86 — APPROVED** [review]
test

File: src/pages/Main.js [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+          onClick={() => {
+            store.disPatch({
+              type: 'PLUS',
+              counter: 1,
+            });
+          }}
+        >
+          클릭
+        </button>
+        <div>1</div>
```
Comment (sstaar91) [inline src/pages/Main.js:27]: 삭제해주세요

**Review by sstaar91 — COMMENTED** [review]
check

**Pull request closed without merging**